### ISSUE TITLE
Sortable Text Option Firefox text input

### ISSUE THREAD

**Issue by leeb003** [reduxframework/redux-framework#537]
Hello,
I've been playing with the framework and it's a nice tool you guys are putting together!
I thought I would bring to your attention a problem I ran across today with the 'Sortable Text Option' using text inputs with Firefox.  Basically text inputs would not work and I found because of line 14 in sortable/field_sortable.js
jQuery( ".redux-sortable" ).disableSelection();
seems to be the culprit, removing this fixes the problem.  I also found jQuery has something about disableSelection usage...
http://api.jqueryui.com/disableSelection/
basically they are saying don't use it.
Thanks for the good work!
Lee
